app/assets/javascripts/modules/map/map.js.coffee

```diff
@@ -72,43 +73,77 @@ ns.Map = do (window,
     #   'geo' (EPSG:4326, WGS 84 / Plate Carree)
     #   'arctic' (EPSG:3413, WGS 84 / NSIDC Sea Ice Polar Stereographic North)
     #   'antarctic' (EPSG:3031, WGS 84 / Antarctic Polar Stereographic)
-    constructor: (el, projection='geo') ->
+    constructor: (el, projection='geo', isMinimap = false) ->
+      @isMinimap = isMinimap
       $(el).data('map', this)
       @layers = []
       map = @map = new L.Map(el, zoomControl: false, attributionControl: false)
 
       map.loadingLayers = 0
-
-      map.addControl(L.control.scale(position: 'topright'))
-
+      if !@isMinimap
+        map.addControl(L.control.scale(position: 'topright'))
+        map.addLayer(new GranuleVisualizationsLayer())
+        map.addLayer(new MouseEventsLayer())
+  
+        map.addControl(new ZoomHome())
+        map.addControl(new ProjectionSwitcher())
+        map.addControl(new SpatialSelection())
+  
+        @legendControl = new LegendControl(position: 'topleft')
+        map.addControl(@legendControl)
+      
       @_buildLayerSwitcher()
-      map.addLayer(new GranuleVisualizationsLayer())
-      map.addLayer(new MouseEventsLayer())
-
-      map.addControl(new ZoomHome())
-      map.addControl(new ProjectionSwitcher())
-      map.addControl(new SpatialSelection())
-
-      @legendControl = new LegendControl(position: 'topleft')
-      map.addControl(@legendControl)
-
       @setProjection(projection)
       @setBaseMap("Blue Marble")
       @setOverlays([OVERLAYS[0], OVERLAYS[2]])
 
-      @time = ko.computed(@_computeTime, this)
-
-      map.fire('edsc.visiblecollectionschange', collections: page.project.visibleCollections())
-      @_granuleVisualizationSubscription = page.project.visibleCollections.subscribe (collections) ->
-        map.fire('edsc.visiblecollectionschange', collections: collections)
+      if !@isMinimap
```

Comment:
A couple of well placed comments could go a long way here. I *think* its just centering the map around the selection spatial param, but that's given little context.

Reply:
I agree with you - though I'm not quite sure of the best comment as this isn't 'new' code (it's only 'new' because it was moved into a conditional).  This code was written quite some time ago.

Reply:
Understandable. I'm seeing the addition of `urlUtil`at the top of the document and then it's used within the code below to. Was this copied from somewhere else that already included that tool?

Reply:
It was - though from where precisely, I do not recall.  If you look at the changes for project_page.js.coffee, you'll see there it uses the @query class to access the URL parameters, which is preferable to using urlUtil.  

However, the map class does not seem to have visibility with the @query object; I attempted to introduce it, and it always comes up empty (i.e., calls to @query.serialize returns an empty structure).  I am not sure of the reason why.  It didn't seem like a huge deal, so I went with urlUtil in this case.  If you might have insight on what the @query class might need in order for it to self-populate (as it is in project_page.js.coffee), I would be happy to work on it further.

Also, if you think the urlUtil stuff could use additional comments to explain things, I'd be happy to do that as well - just let me know ;)

Reply:
Well, that's not a super reassuring response 😬 . Generally I'd ask someone before pushing something that I personally deem 'not the preferred way'. I will admit though, I don't know what the best practices are quite yet so until I cant say much. You are your own quality measure :).

Reply:
Please allow me to attempt more reassurance, then.   :blush:

I think urlUtil is fine, here.  If you do a text search through our files for 'urlUtil', you really will see it used throughout our code.  It's a dependable, structured way to grab parameters from the URL, which is exactly what was needed.

When I first implemented it, I also felt it was fine - however, Bo pointed out early in the PR that it would be preferable (at least to himself) to use the query object.  From that feedback, I updated project_page.js.coffee to use the query object (it had previously also been using urlUtil).  However, the map class doesn't seem to interact with the query object very well, so I've let it using urlUtil.

Bo sometimes was a little hard to read; however, the feeling I got from him was that the use of urlUtil wasn't so bad for him, just more of a preference.  Personally, again - I think it's fine.  I'm absolutely confident in the most important part which is - the code will work and do what it's supposed to.  :)